deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
What do you mean? At the end of this I'd like to see a single Dockerfile that we can use Docker hub to build on commit to master (`canary` tag) & any tag (tagged with appropriate tag).